validator/slashing-protection/local/attester_protect.go

```diff
@@ -90,6 +91,21 @@ func differenceOutsideWeakSubjectivityBounds(latestEpochWritten, targetEpoch uin
 	return latestEpochWritten >= wsPeriod && targetEpoch <= latestEpochWritten-wsPeriod
 }
 
+func isLowerThenMin(
+	minAtt *attestinghistory.MinAttestation,
+	sourceEpoch uint64,
+	targetEpoch uint64,
+) bool {
+	if minAtt == nil {
+		return false
```

Comment:
Perhaps add a comment where why false. To me, I think it's false because if you don't have a min attestation, that probably means the validator has never attested. Is that right?